src/GitHub.VisualStudio/GitHub.VisualStudio.csproj

```diff
@@ -386,6 +392,8 @@
     <Compile Include="Views\Dialog\RepositoryRecloneView.xaml.cs">
       <DependentUpon>RepositoryRecloneView.xaml</DependentUpon>
     </Compile>
+    <Compile Include="Views\GitHubPane\DirectoryIsExpandedToImageMonikerConverter.cs" />
+    <Compile Include="Views\GitHubPane\FileNameToImageMonikerConverter.cs" />
```

Comment:
Where would be an appropriate place to put these converters? I noticed there is a folder with some converters in GitHub.UI but that project doesn't have a lot of dependencies on VS itself (which these do). Also these aren't really general purpose converters like the other ones seem to be, they're specifically made for one view. But this isn't a good place either. Where should I put them?

Reply:
Personally I think that where you've put them is the best place for them: as you say they're tied to a particular view, so it makes sense to put the converters alongside. Not sure if anyone else things differently though?